### PR TITLE
Add keyword difficulty and pla search keywords.  Include filters

### PR DESCRIPTION
**Pull request by michaelaguiar**
Added two new functions, and the ability to use filters.

### CODE REVIEW

**Comment by michaelaguiar** [comment]
Can you tell me why exactly this is failing?

**Comment by andywaite** [comment]
Hey @michaelaguiar just need to update the unit tests for the changes you made.

If you click on details you can see the Travis build log.

screenshot from 2017-01-10 20-05-43

Which explains what's wrong:

screenshot from 2017-01-10 20-06-19
 
You can also run the same tests locally, by installing PHPunit, `composer install`-ing and then running the test suite with `phpunit`.


Anyway, just need to change the 9 in `$this->assertEquals(9, count($types));` to 11 in `test/Data/TypeTest.php` and 39 in `$this->assertEquals(39, count($columns));` to 40 in `test/Data/ColumnTest.php`

To be honest, these unit tests look overkill anyway in retrospect and could possibly be removed.

**Comment by michaelaguiar** [comment]
Ah I see.  Thanks for the insight!  All updated.

File: src/Client.php [hunk 1]
```diff
@@ -182,6 +192,16 @@ public function getDomainAdwordsUnique($domain, $options = [])
      * @param array $options
      * @return ApiResult
      */
+    public function getKeywordDifficulty($phrase, $options = [])
```
Comment (andywaite) [inline src/Client.php]: Need to update docblock above to say $phrase

Comment (andywaite) [inline src/Client.php:195]: Need to update docblock above to say $phrase rather than $domain

**Review by andywaite — CHANGES_REQUESTED** [review]
Minor thing, but one of the docbloc comments has a mistake.  Fix and we can get this merged in :D

**Comment by michaelaguiar** [comment]
Oops!  All updated.  Thanks!

**Review by michaelaguiar — COMMENTED** [review]
Change made.

**Review by andywaite — APPROVED** [review]

**Pull request merged**